config/initializers/secure_headers.rb

```diff
@@ -1,11 +1,9 @@
 # -*- encoding : utf-8 -*-
-::SecureHeaders::Configuration.configure do |config|
+::SecureHeaders::Configuration.default do |config|
 
   # https://tools.ietf.org/html/rfc6797
   if AlaveteliConfiguration::force_ssl
-    config.hsts = { :max_age => 20.years.to_i, :include_subdomains => true }
-  else
-    config.hsts = false
+    config.hsts = "max-age=#{20.years.to_i}; includeSubdomains; preload"
```

Comment:
As far as I can tell, we weren't using the `"preload"` directive, as it's only set [if given the `:preload` key](https://github.com/twitter/secureheaders/blob/2.x/lib/secure_headers/headers/strict_transport_security.rb#L35).

I think we also want to use `SecureHeaders::OPT_OUT` when we're not using SSL, as:

> All `nil` values will fallback to their default values. `SecureHeaders::OPT_OUT` will disable the header entirely.

Since we're not setting `config.hsts` when `force_ssl` is false, I'm assuming we're falling back to the default.